addon/templates/components/uni-datepicker.hbs

```diff
@@ -2,18 +2,36 @@
   center=center
   selected=selected
   onCenterChange=(action onCenter value="moment")
-  onSelect=(action onSelect value="moment") as |calendar|}}
+  onSelect=(action onSelect value="moment")
+  classNames=(if isFullScreen "fullscreen")
+as |calendar|}}
+
+  {{yield}}
+
+  {{#if isFullScreen}}
```

Comment:
I don't think the `uni-datepicker` should be the one "fullscreening": it should be the uni-calendar :/